tests/lib/release-ops.js

```diff
@@ -93,16 +94,18 @@ describe("ReleaseOps", () => {
                 type: "patch",
                 changelog: {
                     fix: [
-                        "* [`5b4812a`](https://github.com/eslint/eslint-release/commit/5b4812a956935358bf6e48f4d75a9bc998b3fe41) Fix: Something (Foo Bar)",
+                        "* [`5b4812a`](https://github.com/eslint/eslint-release/commit/5b4812a956935358bf6e48f4d75a9bc998b3fe41) fix: Something (Foo Bar)",
                         "* [`24b2fdb`](https://github.com/eslint/eslint-release/commit/24b2fdb310b89d7aad134df7e8863a5e055ac63f) Fix: Something else (Foo B. Baz)"
                     ],
                     docs: [
-                        "* [`00a3526`](https://github.com/eslint/eslint-release/commit/00a3526f3a6560e4f91d390725b9a70f5d974f89) Docs: Something else (foobar)"
+                        "* [`00b3526`](https://github.com/eslint/eslint-release/commit/00b3526f3a6560e4f91d390725b9a70f5d974f80) docs: Something else (foobar)",
+                        "* [`00a3526`](https://github.com/eslint/eslint-release/commit/00a3526f3a6560e4f91d390725b9a70f5d974f80) Docs: Something else (foobar)"
```

Comment:
I don't feel like it's a good practice to mix things like `docs` & `Docs`. 

what do you think converting to the same style?  docs/Docs is fine.

Reply:
Since these are directly showing the existing commit messages, I think it's fine to mix. Yes, it's a little awkward, particularly in categories that might have multiple prefixes like `Update:` mixed with `feat:` or `Breaking:` mixed with `fix!:`. That will go away in time once we've fully migrated over to conventional commits, and eventually the conventional commits tooling will be responsible for generating the changelogs. Since this code to map conventional commits back to our current convention for semver purposes is designed to be deleted, I'm okay if it has a couple rough edges during the migration.

Reply:
Yeah, it will be awkward during the transition, but I don’t think it’s worth the added complexity to manipulate the original message.